packages/flutter/lib/src/services/binding.dart

```diff
@@ -90,7 +90,7 @@ mixin ServicesBinding on BindingBase, SchedulerBinding {
   /// [BackgroundIsolateBinaryMessenger.ensureInitialized], which takes a
   /// [RootIsolateToken] as its argument. The value `null` is returned when
   /// executed from background isolates.
-  ui.RootIsolateToken? get rootIsolateToken => ui.RootIsolateToken.instance;
+  static ui.RootIsolateToken? get rootIsolateToken => ui.RootIsolateToken.instance;
```

Comment:
I switched this to static since it doesn't make sense to have a ServiceBinding.instance to call it because we want to call it from background isolates where there will be no ServiceBinding.